src/abstract-interpretation/handler/binop/binop.ts

```diff
@@ -0,0 +1,45 @@
+import {Handler} from '../handler'
+import {AINode} from '../../processor'
+import {BinaryOperatorFlavor, ParentInformation, RBinaryOp} from '../../../r-bridge'
+import {guard} from '../../../util/assert'
+import {operators} from './operators'
+
+export type BinOpOperators = {
+	[key in BinaryOperatorFlavor]: (lhs: AINode, rhs: AINode, node: RBinaryOp<ParentInformation>) => AINode
+}
+
+export class BinOp implements Handler<AINode> {
+	lhs:           AINode | undefined
+	rhs:           AINode | undefined
+	readonly node: RBinaryOp<ParentInformation>
+
+	constructor(node: RBinaryOp<ParentInformation>) {
+		this.node = node
+	}
+
+	getName(): string {
+		return `Bin Op (${this.node.flavor})`
+	}
+
+	enter(): void {
+		console.log(`Entered ${this.getName()}`)
```

Comment:
please switch to the flowr logger (you can create your own sublogger: `log.getSubLogger({ name: this.options.sessionName })`)